src/storage/config.rs

```diff
@@ -44,6 +45,7 @@ impl Default for Config {
             scheduler_concurrency: DEFAULT_SCHED_CONCURRENCY,
             scheduler_worker_pool_size: if total_cpu >= 16 { 8 } else { 4 },
             scheduler_pending_write_threshold: ReadableSize::mb(DEFAULT_SCHED_PENDING_WRITE_MB),
+            reserve_space_for_recover: ReadableSize::gb(DEFAULT_RESERVER_SPACE_SIZE),
```

Comment:
maybe only call it `reserved_space` is enough. The purpose of this config can be documented as the comment

Reply:
Ok.